mono/mini/interp/interp.c

```diff
@@ -3537,37 +3544,18 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 	};
 #endif
 
+	if (method_entry (context, frame,
 #if DEBUG_INTERP
-	debug_enter (frame, &tracing);
-#endif
-
-	MonoException *ex;
-
-	// FIXME Use method_entry here. But it assumes clause_args == NULL.
-
-	if (!frame->imethod->transformed) {
-#if DEBUG_INTERP
-		char *mn = mono_method_full_name (frame->imethod->method, TRUE);
-		g_print ("(%p) Transforming %s\n", mono_thread_internal_current (), mn);
```

Comment:
Why is this removed?

Reply:
It is the same as method_entry. Use method_entry instead.
The switch from imethod to cmethod is part of the point -- that is passing a parameter across blocks.

Reply:
The ifdef is how to minimize repetition.

Reply:
If I may, I'm going to dismiss this.
1. We talked.
2. This PR is split up.
3. This part is in another PR, that is approved.
4. This PR will only have a little left.
5. It isn't approved yet anyway, dismissing this won't make it approved.